src/Views/Pages/Supporters.elm

```diff
@@ -3,8 +3,9 @@
 
 module Views.Pages.Supporters exposing (supportersContent)
 
-import Html exposing (Html, div, h2, p, text)
-import Html.Attributes exposing (class)
+import Assets exposing (AssetPath(..), path)
+import Html exposing (Html, div, h2, p, text, img)
```

Comment:
There is a tool that does this for us (elm-analyse or I thought new version of format would also) - so it won't matter... but they should be in alphabetical order (with Caps ones first) which will make code review/ explicit changes easier to spot in future.

Please change for now manually

Reply:
I think I've understood what you meant and made the changes?